chromium/manifest.json

```diff
@@ -9,7 +9,12 @@
     "background": {
         "scripts": [
             "background-scripts/bootstrap.js",
-            "background-scripts/util.js",
+						"background-scripts/log.js",
+						"background-scripts/validation.js",
+						"background-scripts/modules/ip_utils.js",
+						"background-scripts/modules/browser_session.js",
```

Comment:
It seems that the indents here are off

Reply:
My code editor strikes again. I'll fix that.